src/main/java/org/killbill/billing/plugin/adyen/client/payment/service/AdyenPaymentServiceProviderHostedPaymentPagePort.java

```diff
@@ -100,9 +100,14 @@ public Map getDirectory(final String merchantAccount,
         return directoryClient.getDirectory(params);
     }
 
-    // Used to verify completion
-    @SuppressWarnings("deprecation")
+
     public HppCompletedResult parseAndVerifyRequestIntegrity(final Map<String, String> requestParameterMap) {
+        return parseAndVerifyRequestIntegrity(requestParameterMap, adyenConfigProperties, logger);
+    }
+
+    public static HppCompletedResult parseAndVerifyRequestIntegrity(final Map<String, String> requestParameterMap,
+                                                                    final AdyenConfigProperties adyenConfigProperties,
+                                                                    final Logger logger) {
```

Comment:
Don't pass a `logger`, use the one from `AdyenPaymentServiceProviderHostedPaymentPagePort`.

Reply:
Removed the use of static method. See b08b884